dask/async.py

```diff
@@ -438,6 +433,8 @@ def get_async(apply_async, num_workers, dsk, result, cache=None,
     keyorder = order(dsk)
 
     state = start_state_from_dask(dsk, cache=cache, sortkey=keyorder.get)
+    if callbacks.start:
+        callbacks.start(dsk, state)
```

Comment:
Ah, in that case I'm shamefaced for commenting.  I do <3 incremental committing.